catroid/src/main/java/org/catrobat/catroid/stage/StageListener.java

```diff
@@ -251,7 +251,7 @@ public void create() {
 
 	public void cloneSpriteAndAddToStage(Sprite cloneMe) {
 		Sprite copy = cloneMe.cloneForCloneBrick();
-		copy.resetSprite();
+		//copy.resetSprite(); // can I safely remove this?
```

Comment:
My guess was wrong. The Problem is not a [RuntimeException("No physics action available in non-physics sprite!");](https://github.com/Catrobat/Catroid/blob/b5a9fc8323373031b4d561cba3d69de77d094319/catroid/src/main/java/org/catrobat/catroid/content/ActionFactory.java#L764).
The problem won't be an exception but strange behavior. The created Look would be a normal Look which is not connected to the Physics Engine. So it will behave just like a normal Look. (No collision or forces like gravity).